tests/php/Forms/HTMLEditor/HTMLEditorSanitiserTest.php

```diff
@@ -13,6 +13,12 @@ class HTMLEditorSanitiserTest extends FunctionalTest
 
     public function testSanitisation()
     {
+        // ensure configurable value matches what's used in this test
+        $config = HTMLEditorConfig::get('htmleditorsanitisertest');
+        $sanitiser = Injector::inst()->get(HTMLEditorSanitiser::class, false, [$config]);
```

Comment:
It's best not to use Injector in this case, as we are wanting to test the `HTMLEditorSanitiser` class, not whatever someone might have configured to use instead :)

Reply:
updated